src/pages/VenuePage/VenuePage.tsx

```diff
@@ -63,6 +64,8 @@ const VenuePage: React.FC = () => {
   // const [isAccessDenied, setIsAccessDenied] = useState(false);
 
   const { user, profile } = useUser();
+  const { userLocation } = useWorldUserLocation(user?.uid);
+  const { lastSeenIn: userLastSeenIn } = userLocation ?? {};
```

Comment:
I believe I explicitly chose not to do that, as I want `userLocation` to be `undefined` if it doesn't exist, so as not to 'hide' that fact. This syntax here is just a shorthand method for destructuring.

I could also have written it like this, but I tend to find the pattern overly verbose in most cases (this is a bit of an exception since i'm also renaming the variable, usually it's even shorter still):

```typescript
// Method used here
const { lastSeenIn: userLastSeenIn } = userLocation ?? {};

// Non-destructuring version
 const userLastSeenIn = userLocation?.lastSeenIn;

// ----------

// More canonical example
const { lastSeenIn } = userLocation ?? {}; 

// The more verbose version of it that I really dislike. I basically never want to be typing the same variable name out twice when I could be destructuring it instead.
const lastSeenIn = userLocation?.lastSeenIn; 
```

Reply:
Marking this as resolved based on the above.